mw-webapp/src/component/input/Input.tsx

```diff
@@ -0,0 +1,39 @@
+import {FC, HTMLInputTypeAttribute} from "react";
+import cn from "classnames";
```

Comment:
We don't use classnames in the project right now. Please don't use it too. 
iI you think that the project needs to use classnames, then let's discuss this soon (after joining the group)

Reply:
we are using clsx